tests/unit/Services/FormatTest.php

```diff
@@ -16,7 +16,7 @@
 /**
  * @covers Format
  */
-class FormatTest extends TestCase
+class x extends TestCase
```

Comment:
This change seems unintentional? 🤔

Reply:
Oh. This is a mistake. Thanks for catching.